app/components/chart-pie.js

```diff
@@ -26,37 +27,57 @@ export default Component.extend({
     const width = get(this, 'width');
     const height = get(this, 'height');
     const displayTooltip = get(this, 'displayTooltip');
-    const hideTooltip = get(this, 'hideTooltip');
+    // const hideTooltip = get(this, 'hideTooltip');
```

Comment:
Commented code has been removed. We can wait to merge if you would like to. I will continue working on this branch.